spec/client_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rspec'
+require_relative '../lib/rubyai'
+
+RSpec.describe RubyAI::Client do
+  describe '#mode' do
+    it 'changes the model' do
+      client = RubyAI::Client.new('your_api_key', 'Who am i?')
```

Comment:
We need to mock this data.
We can not use real api_key in the test.

Reply:
it works without entering the key